### PR TITLE
map-size-changed

### PR DESCRIPTION
**Pull request by ItaiOz**
Hi Daniel,

I've pushed the changes, waiting for review

### CODE REVIEW

File: src/components/molecules/map/Map.tsx [hunk 1]
```diff
@@ -27,13 +27,13 @@ interface IProps {
...
 
-const Map: FC<IProps> = ({ zoom = INITIAL_ZOOM, center=INITIAL_CENTER, data, children }) => {
+const Map: FC<IProps> = ({ zoom = INITIAL_ZOOM, center = INITIAL_CENTER, data, children }) => {
   const classes = useStyles();
 
   const bounds = getBounds(data);
 
   return (
-    <MapContainer zoom={zoom} center={center} className={classes.root}>
+    <MapContainer zoom={zoom} center={center} className={classes.root} style={{ height: '90%', alignSelf: 'end' }}>
```
[the remark this answers is not in the document]
Comment (ItaiOz) [reply src/components/molecules/map/Map.tsx]: Done. inline style got replaced by extra class co compnent

File: src/components/molecules/map/Map.tsx [hunk 2]
```diff
@@ -25,15 +26,16 @@ interface IProps {
...
 
-const Map: FC<IProps> = ({ zoom = INITIAL_ZOOM, center=INITIAL_CENTER, data, children }) => {
+const Map: FC<IProps> = ({ zoom = INITIAL_ZOOM, center = INITIAL_CENTER, data, children, customHeightStyle }) => {
   const classes = useStyles();
 
   const bounds = getBounds(data);
 
   return (
-    <MapContainer zoom={zoom} center={center} className={classes.root}>
+    <MapContainer zoom={zoom} center={center} className={`${classes.root} ${customHeightStyle}`}>
```
Comment (danielsh28) [inline src/components/molecules/map/Map.tsx:38]: give the map hight or width of 100%

File: src/components/molecules/widgets/MostSevereAccidentsMapWidget.tsx [hunk 3]
```diff
@@ -32,16 +33,25 @@ const useStyles = makeStyles({
   logoTextSpace: {
     marginInlineStart: '5px',
   },
+  mapCustomHeight: {
+    maxHeight: '90%',
+    alignSelf: 'flex-end',
+  },
+  mapBox: {
+    height: '95%',
```
Comment (danielsh28) [inline src/components/molecules/widgets/MostSevereAccidentsMapWidget.tsx:41]: you can change the height here

**Review by danielsh28 — CHANGES_REQUESTED** [review]

File: src/components/molecules/LocationMap.tsx [hunk 4]
```diff
@@ -1,14 +1,16 @@
...
 import Map from './map/Map';
+import { CSSProperties } from '@material-ui/core/styles/withStyles';
 
 interface IProps {
   items: IPoint[] | any;
+  customHeightStyle?: CSSProperties;
 }
 
-const LocationMap: FC<IProps> = ({ items }) => {
+const LocationMap: FC<IProps> = ({ items, customHeightStyle }) => {
```
Comment (danielsh28) [inline src/components/molecules/LocationMap.tsx:13]: do we still need `customHeightStyle `? if not remove

**Review by danielsh28 — CHANGES_REQUESTED** [review]

**Pull request merged**